keyvaluestore.js

```diff
@@ -6,14 +6,22 @@ app.use(bodyParser.json()); // support json encoded bodies
 app.use(bodyParser.urlencoded({ extended: true })); // support encoded bodies
 
 const port = process.env.MICRO_KEYVALUESTORE_PORT || process.env.PORT || 9000;
+const masterEndpoint = process.env.MICRO_MASTER_ENDPOINT || '';
```

Comment:
I am thinking about simplifying the logic a bit. What if you initialise the `keyValueStore` with these data if it exists and then in `POST` method disallow updating the `keyValueStore`?

Using this `get` will not require any changes, `post` will will have to check that the value does not exist and you can completely remove the mapping.